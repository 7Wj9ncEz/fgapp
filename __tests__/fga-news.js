import 'react-native';
import React from 'react';
import FgaNews from '../js/components/fga-news.js';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <FgaNews
      page={1}
      articles={[]}
      fetchArticles={() => []}
    />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
