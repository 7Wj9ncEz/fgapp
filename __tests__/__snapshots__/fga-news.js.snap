exports[`test renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "yellow",
      "flex": 1,
      "justifyContent": "center",
    }
  }>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail">
    News
  </Text>
</View>
`;
